.github/release.yml

```diff
@@ -0,0 +1,18 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+  categories:
+    - title: Breaking Changes 🛠
+      labels:
+        - breaking
+    - title: New Features 🎉
+      labels:
+        - enhancement
+    - title: Bug Fixes 🐞
+        - bug
+    - title: Documentation 📄
+        - documentation
+    - title: Other Changes
+      labels:
+        - "*"
```

Comment:
For improvement that doesn't have breaking changes (e.g. https://github.com/caraml-dev/merlin/pull/484), will it fall to New Features or Other Changes?

Reply:
My definition is, if the users will benefit from the change, it should be "New Features" - IMO, metrics are in this category.

As for Other Changes, currently, the only PR label that falls under this category is Maintenance - this could be simple dependency updates, code refactoring, etc.

We can refine this list as we go though.